www/src/styles/global.css

```diff
@@ -38,7 +38,7 @@
 
   html {
     scroll-behavior: smooth;
-    @apply scrollbar-thin scrollbar-track-transparent scrollbar-thumb-t3-purple-300 dark:scrollbar-track-transparent dark:scrollbar-thumb-t3-purple-300;
+    @apply scroll-pt-36 scrollbar-thin scrollbar-track-transparent scrollbar-thumb-t3-purple-300 dark:scrollbar-track-transparent dark:scrollbar-thumb-t3-purple-300 lg:scroll-pt-28;
```

Comment:
@c-ehrlich was this the line removed in the other pr? Might cause the shifting?

Reply:
thats almost certainly it, the other PR removed a `scroll-padding-top: 7rem;` from `html`

Reply:
I think for mobile/tablet we have change `scroll-padding-top` because now this section also will take space in header so with the current `padding-top` it is not aligning properly with header when we click from dropdown.
We have to increase scroll value for mobile/tablet.